### DISCUSSION TITLE
optimize the volume of my Next. Js project

### DISCUSSION THREAD

**Discussion by goc-chushixian** [vercel/next.js#39490] [Help] [answered]
I want to optimize the volume of my Next. Js project. Now the image is 800m in size, how can I modify the Webpack configuration to optimize it to 500m

**Comment by ijjk** [comment, marked as answer]
Hi, you may want to checkout our Docker example here https://github.com/vercel/next.js/blob/canary/examples/with-docker/README.md which shows leveraging `output: 'standalone'` for minimizing image sizes.

**Reply by goc-chushixian** [reply]
I'm sorry for the lack of clarity in my question, my question now is how to use Webpack configuration to optimize my project

**Reply by ijjk** [reply]
`webpack` shouldn't be the cause of your image being large, it is most likely due to un-necessary `node_modules` being included in your image which `output: 'standalone'` addresses.

**Reply by goc-chushixian** [reply]
So how do I configure in Next. Config. js to eliminate unnecessary node_modules packages, which I thought the next. Js package would handle automatically

**Reply by goc-chushixian** [reply]
When I finished packing, I found that the files I had packed still depended on @react and @next and other dependencies in node_modules. How do I pack all these dependencies into my.next folder
